### PR TITLE
Fix default contact method for clients

### PR DESCRIPTION
**Pull request by ycedres**
# Description

Change documentation in 4.2, as "classic salt" is not longer the default but now the Salt Bundle is used by default during bootstrapping

# Target branches

Which documentation version does this PR apply to?

- [ ] Master (Default)
- [ ] Manager-4.3
- [X] Manager-4.2

# Links

Fixes https://github.com/SUSE/spacewalk/issues/20019

### CODE REVIEW

File: modules/client-configuration/pages/contact-methods-saltbundle.adoc [hunk 1]
```diff
@@ -27,11 +20,7 @@ The Salt Bundle is provided as the [package]``venv-salt-minion`` that consists o
 Bootstrapping with {webui} is using Salt Bundle as well, so bootstrapping with {webui} is not Python dependant.
 Using the Salt Bundle, it is no longer needed that the client provides any Python interpreter or modules.
 
-Before bootstrapping new client you need to include [package]``venv-salt-minion`` to the bootstrap repository relevant to the client with explicitly specifying [package]``venv-salt-minion`` as an additional package on calling [command]``mgr-create-bootstrap-repo``.
-----
-mgr-create-bootstrap-repo -c DISTRIBUTION_LABEL venv-salt-minion
-----
-You can switch existing clients to the Salt Bundle method.
+If you bootstrap new clients, registration with the Salt Bundle is the default method. You can switch existing clients to the Salt Bundle method.
```
Comment (keichwa) [inline modules/client-configuration/pages/contact-methods-saltbundle.adoc:23]:
```suggestion
If you bootstrap new clients, registration with the Salt Bundle is the default method.
You can switch existing clients to the Salt Bundle method.
```

**Review by keichwa — APPROVED** [review]
Thank you!  Just a cosmetic suggestion (to help adoc with file processing, we use a one-line-per-sentence policy).

File: modules/client-configuration/pages/contact-methods-saltbundle.adoc (hunk 1 above)
Comment (ycedres) [reply modules/client-configuration/pages/contact-methods-saltbundle.adoc:23]: Thanks for letting me know @keichwa!

**Comment by keichwa** [comment]
Yeray, I do not know whether you have merge right.  If yes, feel free to squash and merge, if not we can do it for you.

File: modules/client-configuration/pages/contact-methods-saltbundle.adoc [hunk 2]
```diff
@@ -27,10 +20,7 @@ The Salt Bundle is provided as the [package]``venv-salt-minion`` that consists o
 Bootstrapping with {webui} is using Salt Bundle as well, so bootstrapping with {webui} is not Python dependant.
 Using the Salt Bundle, it is no longer needed that the client provides any Python interpreter or modules.
 
-Before bootstrapping new client you need to include [package]``venv-salt-minion`` to the bootstrap repository relevant to the client with explicitly specifying [package]``venv-salt-minion`` as an additional package on calling [command]``mgr-create-bootstrap-repo``.
-----
-mgr-create-bootstrap-repo -c DISTRIBUTION_LABEL venv-salt-minion
-----
+If you bootstrap new clients, registration with the Salt Bundle is the default method.
```
Comment (meaksh) [inline modules/client-configuration/pages/contact-methods-saltbundle.adoc:23]: I would put this as an `[IMPORTANT]` note, to make it notice that the "Salt Bundle has become the default method in 4.2, maybe we should put the new IMPORTANT note at line 15 of this page, where you removed the previous one.

Then here, we can simply reiterate this in a normal sentenace like the this one you added.

Comment (ycedres) [reply modules/client-configuration/pages/contact-methods-saltbundle.adoc:27]: I added the `[IMPORTANT]` note as suggested.

**Comment by ycedres** [comment]
> Yeray, I do not know whether you have merge right. If yes, feel free to squash and merge, if not we can do it for you.

@keichwa thank you for committing the change on the one-line-per-sentence policy. In fact, I believe I cannot merge the branch, could you do it for me if you don't mind? Thanks!

**Review by meaksh — APPROVED** [review]
LGTM

**Review by keichwa — APPROVED** [review]

**Pull request merged**